lms/djangoapps/grades/models.py

```diff
@@ -34,60 +33,62 @@ class BlockRecordList(tuple):
     An immutable ordered list of BlockRecord objects.
     """
 
-    def __new__(cls, blocks):
+    def __new__(cls, blocks, course_key):  # pylint: disable=unused-argument
         return super(BlockRecordList, cls).__new__(cls, tuple(blocks))
```

Comment:
Python requires overriding the `__new__` operator when subclassing from an immutable type, such as `tuple`.  It requires this in order to allocate appropriate space for the immutable object.  See http://stackoverflow.com/questions/674304/pythons-use-of-new-and-init/674369#674369.

And subsequently, it also requires the signature of the `__init__` method to match the signature of the `__new__` method.